build.sbt

```diff
@@ -27,22 +29,68 @@ val core = crossProject.crossType(CrossType.Pure)
   .settings(Dependencies.acyclic)
   .settings(Dependencies.catsCore)
   .settings(Dependencies.scalaCheck)
-  .configureCross(Coverage.disableScoverage210Jvm)
-  .configureCross(Coverage.disableScoverage210Js)
   .jsSettings(Settings.commonJsSettings:_*)
   .jvmSettings(Settings.commonJvmSettings:_*)
 
 val coreJVM = core.jvm
 val coreJS = core.js
 
+lazy val docsMappingsAPIDir = settingKey[String]("Name of subdirectory in site target directory for api docs")
+
+lazy val docSettings = Seq(
+  micrositeName := "Cats MTL",
+  micrositeDescription := "Companion library to cats providing monad transformers",
+  micrositeAuthor := "Typelevel contributors",
+  micrositeHighlightTheme := "atom-one-light",
+  //  micrositeHomepage := "http://typelevel.org/cats",
+  micrositeBaseUrl := "cats-mtl",
+  micrositeDocumentationUrl := "api",
+  micrositeGithubOwner := "edmundnoble",
```

Comment:
Might as well adjust this to the same as cats: `micrositeGithubOwner := "typelevel"`